google-cloud-examples/src/main/java/com/google/cloud/examples/compute/v1/ComputeInstanceExample.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright 2019 Google LLC.  All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.compute.v1;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.AccessConfig;
+import com.google.api.services.compute.model.AttachedDisk;
+import com.google.api.services.compute.model.AttachedDiskInitializeParams;
+import com.google.api.services.compute.model.Instance;
+import com.google.api.services.compute.model.InstanceAggregatedList;
+import com.google.api.services.compute.model.InstanceList;
+import com.google.api.services.compute.model.InstancesScopedList;
+import com.google.api.services.compute.model.NetworkInterface;
+import com.google.api.services.compute.model.Operation;
+import com.google.cloud.compute.v1.ProjectZoneOperationName;
+import com.google.cloud.compute.v1.ZoneOperationClient;
+import com.google.common.collect.Lists;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * Sample to demo using Google Compute Engine instances using Java and the Google Compute Engine
```

Comment:
Needs to rephrase. Are you trying to demonstrate how to use Google Compute Engine instances by using Java and the Google Compute Engine APIs? 
Suggestions: 
1. The sample demonstrates Google Compute Engine instances in Java by using Google Compute Engine API. 
2. The sample demonstrates how to use Google Compute Engine instances by using Java and the Google Compute Engine API.